components/server/src/auth/generic-auth-provider.ts

```diff
@@ -466,6 +475,22 @@ export class GenericAuthProvider implements AuthProvider {
                     ...defaultLogPayload,
                 });
 
+                try {
```

Comment:
A comment, or even a method, would be nice, to call out what the purpose of this piece of code is.

Reply:
done